#! /usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import int_t
from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator


PACKAGE = 'mirror_recognition'

gen = ParameterGenerator()

gen.add('scene_idx', int_t, 0, 'Index of scene.', 0, 0, 10000)

exit(gen.generate(PACKAGE, "mirror_recognition", "PublishRawDataset"))
